classes/ProductAssembler.php

```diff
@@ -67,25 +67,29 @@ private function addMissingProductFields(array $rawProduct)
 
         $sql = "SELECT
                     p.*,
+                    ps.*,
                     pl.*,
                     sa.out_of_stock,
                     IFNULL(sa.quantity, 0) as quantity,
                     (DATEDIFF(
-				p.`date_add`,
-				DATE_SUB(
-					'$now',
-					INTERVAL $nbDaysNewProduct DAY
-				)
-			) > 0) as new
+                        p.`date_add`,
+                        DATE_SUB(
+                            '$now',
+                            INTERVAL $nbDaysNewProduct DAY
+                        )
+                    ) > 0) as new
                 FROM {$prefix}product p
                 LEFT JOIN {$prefix}product_lang pl
                     ON pl.id_product = p.id_product
                     AND pl.id_shop = $idShop
                     AND pl.id_lang = $idLang
                 LEFT JOIN {$prefix}stock_available sa
-			        ON sa.id_product = p.id_product 
+			        ON sa.id_product = p.id_product
 			        AND sa.id_shop = $idShop
-			    WHERE p.id_product = $idProduct LIMIT 1";
+                LEFT JOIN {$prefix}product_shop ps
+			        ON ps.id_product = p.id_product
+			        AND ps.id_shop = $idShop
+			    WHERE p.id_product = $idProduct";
```

Comment:
```suggestion
			    WHERE p.id_product = $idProduct
			    LIMIT 1";
```

Reply:
The limit 1 is not needed because it's selecting from ps_product with a specific id_product. I don't think it's necessary

Reply:
Done